### PR TITLE
Add `load_policy` to `mc-sgx-dcap-ql`

### PR DESCRIPTION
**Pull request by nick-mobilecoin**

### CODE REVIEW

**Comment by nick-mobilecoin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #147** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/147" target="_blank">Graphite</a> 
    * **PR #148** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/148" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/148?utm_source=stack-comment).

**Review by awygle — APPROVED** [review]

**Review by NotGyro — APPROVED** [review]
Looks good to me!

**Comment by nick-mobilecoin** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/148?utm_source=stack-comment).


Job Reference: EQRIYxUG81AqqoPYAu7h

**Pull request merged**
